app/policies/partner_policy.rb

```diff
@@ -0,0 +1,25 @@
+# Authorization for CRUD actions on Partner using Pundit
+#
+# Pundit relies on current_user
+#   see `pundit_user` method in partners_controller
+#
+# Partners can only affect their own records.
+# Later may want to admins authorization.
+class PartnerPolicy < ApplicationPolicy
+  def create?
```

Comment:
I think you inclination on the delete is correct and probably create too. Since partners are created via api call I don't think we need the create policy. I also imagine for reporting purposes we probably don't want deletes -- just soft deletes like oyu suggest.